src/main/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParameters.java

```diff
@@ -79,6 +79,12 @@ public Javadoc visitDocComment(Javadoc.DocComment javadoc, ExecutionContext ctx)
                     boolean useNewBody = false;
 
                     List<Javadoc> body = javadoc.getBody();
+                    // To fix elements on the first line without space
+                    if (!body.isEmpty() && !(body.get(0) instanceof Javadoc.LineBreak)) {
+                        body = new ArrayList<>(body);
+                        body.add(0, null); // we can use null since this element is never going to be used.
+                    }
```

Comment:
by adding this trailing element, we can always process the elements in this loop, and we no longer require the specific element visitors, that were causing a bit of confusion and complexity to debug the recipe. Now those are gone and changed to checking boolean methods (isEmpty...)